card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -79,13 +78,19 @@ class CardComponent private constructor(
 
     override fun onInputDataChanged(inputData: CardInputData): CardOutputData {
         Logger.v(TAG, "onInputDataChanged")
+        if (!isStoredPaymentMethod()) {
+            filteredSupportedCards = updateSupportedFilterCards(inputData.cardNumber)
+        }
+        val cardDelegate = mPaymentMethodDelegate as CardDelegate
+        val firstCardType: CardType? = if (filteredSupportedCards.isNotEmpty()) filteredSupportedCards[0] else null
+
         return CardOutputData(
-            validateCardNumber(inputData.cardNumber),
-            validateExpiryDate(inputData.expiryDate),
-            validateSecurityCode(inputData.securityCode),
+            cardDelegate.validateCardNumber(inputData.cardNumber),
+            cardDelegate.validateExpiryDate(inputData.expiryDate),
+            cardDelegate.validateSecurityCode(inputData.securityCode, configuration, firstCardType),
             validateHolderName(inputData.holderName),
```

Comment:
Can this be `cardDelegate.validateHolderName` instead?

Reply:
Yeah, this was not supposed to be part of this PR 😬 . Maybe I can finish this part and add it as well.